fix: set keyboard specific label font if specified

ermshiperete opened the pull request:
Fixes: https://github.com/keymanapp/keyman/issues/12019

Testing will be done in done keymanapp/keyman#12277.

@keymanapp-test-bot skip

ermshiperete committed:
test build

ermshiperete committed:
Set keyboard specific label font if specified

Fixes: keymanapp/keyman#12019

mcdurdin commented on Onboard/OnboardGtk.py:
```diff
@@ -568,6 +568,8 @@ def cb_keys_changed(self, keymap):
     def cb_keyman_changed(self, name):
         """ keyman keyboard change """
         print("keyman changed to", name)
+        # TODO: why is 'None' passed as a string here???
```
Do we need to track this?

ermshiperete replied:
https://github.com/keymanapp/keyman/issues/12330

ermshiperete committed:
Release 1.5.0

ermshiperete committed:
fix: add missing dependency and fix tests

ermshiperete committed:
Release 1.5.2

Increase version to bring it even with Focal version.

ermshiperete committed:
Release 1.5.3

We need to use different version numbers for regular package build
and Focal package build because they don't share the same orig.tar.gz
package.
